<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomerAddDialog</class>
 <widget class="QDialog" name="CustomerAddDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>613</width>
    <height>522</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>613</width>
    <height>522</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>613</width>
    <height>522</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>会员名</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>会员密码</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>联系方式</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="NameEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>69</y>
     <width>431</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="PasswdEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>180</y>
     <width>431</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="ContanctEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>290</y>
     <width>431</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>410</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
